### ISSUE TITLE
Add French translation for chat command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#90]
```
Todo: add french translation for chat command
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 8 Feb 2011 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
this changeset needs also some attention: 
http://code.google.com/p/gtalksms/source/detail?r=21bd50c5d4b0023850513599f4af6b
30c517dfa8
```

Original comment by `fschm...@gmail.com` on 11 Feb 2011 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
